File: Web.Admin/src/api/services/config/models/QueryConfigPagedOutput.ts
/**
 * Fast.Center.Service.Config.Dto.QueryConfigPagedOutput 获取配置分页列表输出
 */
export interface QueryConfigPagedOutput {
  /**
   * 配置Id
   */
  configId?: number;
  /**
   * 配置编码
   */
  configCode?: string;
  /**
   * 配置名称
   */
  configName?: string;
  /**
   * 配置值
   */
  configValue?: string;
  /**
   * 备注
   */
  remark?: string;
  /**
   * 创建者用户名称
   */
  createdUserName?: string;
  /**
   * 创建时间
   */
  createdTime?: Date;
  /**
   * 更新者用户名称
   */
  updatedUserName?: string;
  /**
   * 更新时间
   */
  updatedTime?: Date;
}

File: Api.Server/src/Services/Center/Config/Dto/QueryConfigPagedOutput.cs
namespace Fast.Center.Service.Config.Dto;

/// <summary>
/// <see cref="QueryConfigPagedOutput"/> 获取配置分页列表输出
/// </summary>
public class QueryConfigPagedOutput : PagedOutput
{
    /// <summary>
    /// 配置Id
    /// </summary>
    public long ConfigId { get; set; }

    /// <summary>
    /// 配置编码
    /// </summary>
    public string ConfigCode { get; set; }

    /// <summary>
    /// 配置名称
    /// </summary>
    public string ConfigName { get; set; }

    /// <summary>
    /// 配置值
    /// </summary>
    /// <remarks>
    /// <para>Boolean：[True, False]</para>
    /// </remarks>
    public string ConfigValue { get; set; }

    /// <summary>
    /// 备注
    /// </summary>
    public string Remark { get; set; }
}